combat-trainer.lic

```diff
@@ -557,6 +578,26 @@ class LootProcess
     echo "Last ritual performed: #{@last_ritual}" if $debug_mode_ct
   end
 
+  def butcher_corpse(mob_noun, ritual, game_state)
+    return unless ritual.eql?('butcher')
+    echo "Butchering the #{mob_noun}'s corpse!" if $debug_mode_ct
+    echo " Only butchering it once!" if $debug_mode_ct && butcher_once
+
+    bput("perform preserve on #{mob_noun}", @rituals['preserve'], @rituals['failures'])
+    @equipment_manager.stow_weapon(game_state.weapon_name)
+
+    loop do
+      perform_message = "perform #{ritual} on #{mob_noun}"
+      result = bput(perform_message, @rituals['butcher'], @rituals['failures'])
+      break if result =~ /too battered and beat up/i
+      bput("drop my #{right_hand}", 'You drop', 'You discard', 'Please rephrase')
```

Comment:
Is there a common noun we could use (like `material`?) instead of `right_hand`?

Reply:
No, and it varies from butchery to butchery. It can be one of any body part, so right_hand works the best. Is this not good practice? This seems like the best way to do this, or anything for that matter.

Reply:
The items are something like:

> kobold head
> kobold arm
> kobold leg
> kobold torso

Reply:
I usually prefer something uniquely 'butchery' to avoid accidentally dropping something important in case we get into a bad state. I'm less concerned when the action is stow, for example.

`right_hand` is fine, though.

Reply:
Right, but it can be too many different items. I can check if right_hand contains one of the body part nouns, and then stow the item if not? What do you think?

Reply:
I wouldn't worry about it